sbin/common/common.sh

```diff
@@ -243,3 +243,13 @@ function isFromJdk21LTS() {
   [[ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -ge 21 ]] && [[ $(((BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]-21) % 4)) == 0 ]]
 }
 
+# waits N second (10 by default), printing countdown every second.
+function verboseSleep() {
+  if [[ -z "${1}" ]] ; then
+    local i=10
+  else
+    local i="${1}"
+  fi
+  while [ $i -gt 0 ] ; do echo -n " $i " && sleep 1s && i=$(($i-1)) ; done && echo " $i"
```

Comment:
`sleep 1s` won't work on Solaris so remove the `s`.
```suggestion
  while [ $i -gt 0 ] ; do echo -n " $i " && sleep 1 && i=$(($i-1)) ; done && echo " $i"
```

Reply:
I actually took the `s` from one of temurin-build script, I was sure that defaul it is second for eternity... Thanx for confirming it :)

Reply:
Interesting - I might stick a PR in to change it everywhere else then ;-)